DRAFT_RELEASE_NOTES.md

```diff
@@ -77,33 +58,53 @@ To deactivate any of these migrations, add the following configuration to the `b
 ## Precondition Steps - Required for All Projects
 
 ### Beginning the Upgrade
-To start your aiSSEMBLE upgrade, update your project's pom.xml to use the 1.8.0 version of the build-parent:
+To start your aiSSEMBLE upgrade, update your project's pom.xml to use the 1.9.0 version of the build-parent:
    ```xml
    <parent>
        <groupId>com.boozallen.aissemble</groupId>
        <artifactId>build-parent</artifactId>
-       <version>1.8.0</version>
+       <version>1.9.0</version>
    </parent>
    ```
 
 ## Conditional Steps
-### Upgrade Steps for Projects Leveraging aiSSEMBLE Inference Pipeline
-With the enablement of linting, there is a known issue with the Pylint package when linting on Python modules using the Pydantic package or Python files generated by Protobuff. As a result, when linting is enabled, the project build will error. 
-
-To resolve this error, within the `<project-name>-pipelines/<pipeline-name>/<inference-step-name>/pyproject.toml` file, add the configuration:
-```
-[tool.pylint.'MESSAGES CONTROL']
-extension-pkg-whitelist = "pydantic"
 
-[tool.pylint.MASTER]
-ignore-patterns = '.*pb2[\S]*.py'
+### Projects With Data Delivery Pipelines
+#### Migrate to the V2 Spark Infrastructure Helm Chart
+This version of aiSSEMBLE brings a required upgrade to our new spark-infrastructure helm chart.  This singular
+chart, following aiSSEMBLE's v2 structure, replaces the v1 charts for spark-infrastructure, hive-metastore-db, and 
+hive-metastore-service.  For more information on the chart itself, see the associated [README files](https://github.com/boozallen/aissemble/tree/dev/extensions/extensions-helm/extensions-helm-spark-infrastructure). 
+
+In order to perform this migration, the following steps should be taken: 
+
+- Identify and remove the `fermenter-mda` executions from `<YOUR_PROJECT>-deploy/pom.xml` for 
+the following profiles:
+  - `hive-metastore-service-deploy`
+  - `hive-metastore-db-deploy`
+- Within the associated `fermenter-mda` execution in `<YOUR_PROJECT>-deploy/pom.xml`, replace references to the
+`aissemble-spark-infrastructure-deploy` profile with `aissemble-spark-infrastructure-deploy-v2`
+- Remove references to `hive-metastore-db` and `hive-metastore-service` from your Tiltfile
+- Remove or rename the directory `<YOUR_PROJECT>-deploy/src/main/resources/apps/spark-infrastructure`
```

Comment:
Do they also need to remove/rename the `<YOUR_PROJECT>-deploy/src/main/resources/apps/hive-metastore-service` and `<YOUR_PROJECT>-deploy/src/main/resources/apps/hive-metastore-db`?

Reply:
Actually, they do also need to remove the ArgoCD files for them-- Making that change now.